drivers/mtd/Kconfig

```diff
@@ -23,6 +23,14 @@ config HAVE_MTD_NATIVE
     help
         Indicates that a native MTD is present.
 
+config HAVE_MTD_SDCARD_DEFAULT
+    bool
+    imply MODULE_MTD_SDCARD if MODULE_MTD
```

Comment:
`imply` feels like not to be right here. AFAIK, `imply` is the pendant to `DEFAULT_MODULE` in makefiles.

Reply:
I just copied that from `HAVE_MTD_SDCARD`

Reply:
Ok, so it should be ok.